pkg/imgpkg/v1/copy.go

```diff
@@ -17,61 +17,57 @@ import (
 	regname "github.com/google/go-containerregistry/pkg/name"
 )
 
-type SignatureRetriever interface {
-	Fetch(images *imageset.UnprocessedImageRefs) (*imageset.UnprocessedImageRefs, error)
-}
+const rootBundleLabelKey string = "dev.carvel.imgpkg.copy.root-bundle"
```

Comment:
That is a visualization problem in GitHub :( 
Because git considers the file a move, it only shows in the diff the line changes that are different from the original file. This const is used in 4 places inside the v1/copy.go file